dev.yml

```diff
@@ -1,7 +1,7 @@
 ---
 name: krane
 up:
-  - ruby: 2.5.7 # Matches gemspec
+  - ruby: 3.0.0 # Matches gemspec
```

Comment:
This should be 2.6.6

Reply:
yes.. was using it for testing. Will set it to 2.6.6